### ISSUE TITLE
Can't remove a sideloaded my test add-in

### ISSUE THREAD

**Issue by zjmchoi** [OfficeDev/office-js#3605]
# Provide required information needed to triage your issue
In Window Outlook > Get add-ins > My add-ins > Custom Addins, I can't remove a sideloaded my test add-in because "Remove" menu is disable.

## Your Environment

* Platform [PC desktop, Mac, iOS, Office on the web]: PC desktop
* Host [Excel, Word, PowerPoint, etc.]: Outlook
* Office version number: Microsoft 365 MSO(version 2307 build 16.0.16626.20170) 32bit
* Operating System: Windows 10 Pro (64bit)
 
## Expected behavior

I should be able to remove my test add-in.

## Current behavior

I am testing my outlook add-in on web and app(window) with the same Microsoft account.
Then, at some point, I can't remove a sideloaded my test add-in from app(window).
(but, on web, I can remove a sideloaded it.)
If I install manifest.xml for another test add-in again through "Add from file" menu, my test add-in is still present and I cannot remove it.
It appears that something remains in the cache and is displayed.

## Steps to reproduce

1. ______
2. ______
3. ______

## Link to live example(s)

1. ______
2. ______
3. ______

# Provide additional details

1. ______
2. ______
3. ______

## Useful logs

- [ ] Console errors
- [ ] Screenshots
- [ ] Test file (if only happens on a particular file)

Thank you for taking the time to report an issue. Our triage team will respond to you in less than 72 hours. Normally, response time is <10 hours Monday through Friday. We do not triage on weekends.

**Comment by zjmchoi** [comment]
To help, I provide additional information.
1. Only certain test add-ins have problems, other test add-ins do not.
2. Please refer to the attached file.
[office_issue_report_3605.pptx](https://github.com/OfficeDev/office-js/files/12415127/office_issue_report_3605.pptx)

**Comment by zjmchoi** [comment]
Hi,

It seems that the problem occurred while testing alternately on the web and on the app.

Now it works fine.

Pleas close this issue.

Thanks.

**Comment by Suntechnick** [comment]
We are experiencing the same issue for multiple scenarios among the team.
@zjmchoi can you please give us a hint what could possibly help with the issue?

**Comment by Lesturb** [comment]
We are experiencing the same behavior, add-in is installed and somehow can't be removed, the "remove" is greyed out.
This occurs on multiple add-ins (manifest.xml) / multiple versions of our plug-in.

My guess is that it started behaving weird since last week. 

**Some extra behavior description:**
When the add-in is present, but remove button is greyed out, it can't be removed.
When you install the same manifest.xml the button turns normal again, but after restart it's greyed out again, same goes if you actually remove it after that, it still stays present with the greyed out button.

The only solution that I was able to apply is doing a full restore to a snapshot of my machine, but then I tested on a different machine and the same occurred. 

@zjmchoi if there were any steps you took to solve this, please mention them. We've tried everything from cleaning all caches, reinstalling outlook.

**Comment by zhngx1** [comment]
@Suntechnick and @Lesturb can you provide more information of 
Platform [PC desktop, Mac, iOS, Office on the web]
Host [Excel, Word, PowerPoint, etc.]
Office version number
Operating System

Did you both try to clear the cache following our documentation and still running into the issue?
https://learn.microsoft.com/en-us/office/dev/add-ins/testing/clear-cache#clear-the-office-cache-on-windows

**Comment by zjmchoi** [comment]
Yes, I cleared the cache according to the MS documentation, but the issue still occurred at that time.

Now it works fine, but I don't know the exact cause.

But, as @Lesturb said, it seems to happen when tested using different platforms (web/app on window and mac) and different versions (manifest.xml) with the same ms account.

Deleting the manifest file that was installed on multiple platforms (connected to the same account) was the only action I took.
